docs/toolkit/ansible-avalanche-collection/tutorials/contract_deployement.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 11
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Smart Contract Deployement
+
+In this section, we will deploy a smart contract to launch the ASHToken ERC20 token on an Evm subnet deployed using the ash Ansible collection.
+
+:::note
+For this tutorial, we will use the [`local`](https://github.com/AshAvalanche/ansible-avalanche-getting-started/tree/main/inventories/local) inventory provided by the [Ansible Avalanche Getting Started](https://github.com/AshAvalanche/ansible-avalanche-getting-started) repository. See [Create a Local Test Network](./local-test-network) for details.
+
+We should also have created a Subnet and a blockchain, see [Subnet Creation](/docs/toolkit/ansible-avalanche-collection/tutorials/subnet-creation) for details.
+:::
+
+## Configure the environement
+
+For this tutorial, we will use the [Foundry suit](https://github.com/foundry-rs/foundry) which can be installed through their [tutorial](https://book.getfoundry.sh/getting-started/installation).
+To compile the contract, we need to clone two dependencies into the "ash_token/lib" folder.
+
+```bash title="Command"
+git clone https://github.com/foundry-rs/forge-std
+git clone https://github.com/OpenZeppelin/openzeppelin-contracts.git
```

Comment:
```suggestion
git clone https://github.com/foundry-rs/forge-std lib/forge-std
git clone https://github.com/OpenZeppelin/openzeppelin-contracts.git lib/openzeppelin-contracts
```

Reply:
As an improvement these repos could also be git submodules but we can implement this later.